### PR TITLE
Add last per_hop padding

### PR DESCRIPTION
**Pull request by OrfeasLitos**
Also clarify that payload := `per_hop`

### CODE REVIEW

File: 04-onion-routing.md [hunk 1]
```diff
@@ -289,10 +289,11 @@ using an alternate channel.
 
 ## Payload for the Last Node
 
-When building the route, the origin node MUST use a payload for
-the final node with the following values:
+When building the route, the origin node MUST fill the `per_hop` field for the
+final node with the following values:
 * `outgoing_cltv_value`: set to the final expiry specified by the recipient
 * `amt_to_forward`: set to the final amount specified by the recipient
+* `padding`: set to 20 zero bytes
```
Comment (rustyrussell) [inline 04-onion-routing.md]: `padding` is only 12 bytes, though?

The short_channel_id field is left unspecified, but c-lightning sets it to 0.

Comment (OrfeasLitos) [reply 04-onion-routing.md]: Fixed it

**Comment by OrfeasLitos** [comment]
Ready to merge

**Comment by t-bast** [comment]
Closing this as variable-length onion made it obsolete.

**Pull request closed without merging**